app/src/main/index.ts

```diff
@@ -57,6 +58,37 @@ function createWindow(): void {
   }
 }
 
+function spawnFetchWorker(): Worker {
+  const scriptPath = join(
+    app.getAppPath(),
+    "src",
+    "main",
+    "fetch",
+    "worker.ts",
+  );
+  const worker = new Worker(scriptPath, {
+    execArgv: ["--require", "tsx"],
```

Comment:
Coming from a place where I don't really know how nodejs workers work, I'm curious why it requires using tsx; can vite/typescript not compile it to javascript for us to run?